pkg/frontend/kubeactions/mustgather.go

```diff
@@ -79,6 +79,11 @@ func (ka *kubeactions) MustGather(ctx context.Context, oc *api.OpenShiftCluster,
 		}
 	}()
 
+	latest, err := version.GetLatest()
```

Comment:
maybe add a `// TODO` that it would be best to try to get the cluster y-version and fall back to latest if we can't